packages/service-library/src/servicelib/fastapi/long_running_tasks/_context_manager.py

```diff
@@ -115,10 +115,9 @@ async def _wait_task_completion() -> None:
 
         yield result
     except asyncio.TimeoutError as e:
-        task_removed: bool = await client.cancel_and_delete_task(task_id)
+        await client.cancel_and_delete_task(task_id)
```

Comment:
Previously this was not raising an error. I would try catch the errors here to maintain the shame behaviour.

Reply:
this is a call to the rest API. so in effect if the task does not exist, then a 404 will raise. but I also see that if a timeout comes you also raise up. what happens then?

Reply:
actually no ones is catching that timeout error as well right?

Reply:
ok I made it the same as before

Reply:
but to be honest, I think there are some weird things

Reply:
So this is what I'm thinking here:
- I am fine with raising if a timeout comes or if the task is missing. It means something happened and we might want to investigate
- Since there was timeout above, I'd like to do some cleanup and raise. I want to still "try and signal the server to remove the task from tracking if I can". At this point if the server is not reachable I do not care. So any errors from the cancel_and_delete_task should be ignored. I think it is most important for the client to raise `TaskClientTimeoutError`.

Reply:
I think what I want to say is that this `TaskClientTimeoutError` should have a higher priority in this context than an error for `client.cancel_and_delete_task`.
Maybe we could try catch whatever comes out of `client.cancel_and_delete_task ` and then continue raising the `TaskClientTimeoutError `

Reply:
well I guess it depends a bit on the use case... for now I set it as it was done before.

Reply:
OK, let' us have it like this for now.